data/human/human missions.txt

```diff
@@ -4965,3 +4965,185 @@ mission "Riley"
+				to display
+					has "Sheragi Archaeology: The Emerald Sword 3: done"
+			`	After some time, Sophie calls you both to come to look at the tablet.`
+			label sword-scene
+			scene "scene/sword dome"
+			`	Approaching the tablet, there is a short message written on it:`
+			`			"My dear children. The galaxy runs far and wide, and I have traveled many places, more than most in many lifetimes. I have seen many worlds, both alien and human, too. And it was on those worlds that I have been exploring, discovering, and researching everything you see around you. It was my life's duty.`
+			`			But still, there were many paths I did not take, many rocks I have left unturned, and many worlds I have not yet landed on. And I suspect that you will have read this before those destinations were reached. Well, I have arrived at a different destination now, the only one nobody can understand, but everybody reaches eventually. Do not fear for me.`
+			`			This room you are in now is the culmination of all my work. I hope you understand that it was for your sake that I kept these a secret. My journey was long and unforgiving, but it was my life. I hope you continue that journey, to understand the universe a little better.`
+			`			Your father."`
+			`	Understanding the personal nature of the message, you leave the two of them alone for a bit, walking around and looking at the artifacts on the left of the dome.`
+			`	After some time, the siblings call you back to them. Sophie starts, "Captain <last>, Liam and I have thought through about this, and we think it would be a shame if we just kept this sword as a memory to him. It needs to be used. It was our father's one and only weapon, and it has been many places. Take it to many more." She pulls the sword from the pedestal and hands it to you. "It is yours now."`
+			choice
+				`	"I would be more than grateful to accept it."`
+						goto accepted
+				`	"This is too much for me to take."`
+						goto declined
+			label declined
+			`	"Are you sure?" Liam asks. "This is a one time offer. It would be wasted on us."`
+			choice
+				`	"Yes, I'm sure."`
+				`	"Well, if you frame it that way, I will take it."`
+					goto accepted
+			action
+				payment 50000
+			`	"I see. Please, at least take this extra payment of <credits>," Liam says. "It is for you traveling so much with us."`
+			label accepted
+			action
+				outfit "Old Man's Sword" 1
+			label end
+			action
+				payment 50000
+			`	"Well, it seems like this is the end of our short journey together, Captain <last>," Sophie says.`
+			`	"It was a pleasure to travel with you," adds Liam. "We will always thank you for helping us with our father's will, we never imagined that we would come so far and we would have never done it without you. It means so much to us. We fare you well among the stars!"`
+			`	After he hands you a credit ship of <payment>, you say goodbye to them and leave them in their father's dome.`
+			decline
```

Comment:
```suggestion
			`	After he hands you a credit ship for <payment>, you say goodbye to them and leave them in their father's dome.`
				decline
```
Strangely this decline was shown in the conversation screen. Here's a screenshot: 
![Screenshot_20221219_111712](https://user-images.githubusercontent.com/115441627/208322301-285bccb2-c582-43e1-b795-bb7a0608780a.png)
Maybe the decline isn't needed.

Reply:
hmmm